scripts/index-demo-page.js

```diff
@@ -8,14 +8,23 @@
 
   var getInputValue = function(el) {
     if (el.type === 'url' || el.type === 'text') {
-      return el.value;
+      return encodeURIComponent(el.value);
```

Comment:
types with a `+` such as dash would save correctly but on page reload they wouldn't work. we need to encode and decode uri components.